nautobot/utilities/tests/test_filters.py

```diff
@@ -207,48 +208,55 @@ def test_filter_null(self):
 
         kwargs = {"power_panels": [settings.FILTERS_NULL_CHOICE_VALUE]}
         qs = self.SiteFilterSet(kwargs, self.queryset).qs
+        qs = qs.filter(pk__in=self.site_pk_list)
```

Comment:
This also seems wrong - we shouldn't be changing what the filterset evaluates to, or we're not testing the filterset.